snippets/Responsive-Nuemorphism-Card-Design.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Responsive Nuemorphism Card Design
+tags: interactivity,layout
+---
+
+Creates an awesome nuemorphism effect with cool interactivity to the users. 
+
+- Uses `display: flex` property along with `justify-content: center; align-items: center;` to make the snippet responsive.
+- Creates Nuemorphism effect by using nuemorphism colors like `rgba(255, 255, 255, 0.1)` or `rgba(0, 0, 0, 0.6)` and using nuemorphism shadows. 
+- Uses `:hover` pseudo-selector to make the transitions and interactivity.
+
+```html
+<div class="cont flex">
+    <div class="container flex">
+        <div class="card">
+            <div class="box flex">
+                <div class="content">
+                    <h2>01</h2>
+                    <h3>Card Title</h3>
+                    <p>Lorem ipsum dolor sit amet constetur, adipisicing elit. Soluta, atque!</p>
+                    <a href="#">Read More</a>
+                </div>
+            </div>
+        </div>
+    </div>
+</div>
+```
+
+```css
+.cont {
+  height: 100vh;
+  width: 100%;
+  font-family: "Comic Sans MS", cursive, sans-serif;
```

Comment:
No reason to set a font in general. The snippet should showcase the neumorphism technique alone.